module05/test_example.py

```diff
@@ -4,30 +4,31 @@
 class ExampleTestCase(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
-        print "setUpClass executes only once. CREATE DATABASE;"
+        print('setUpClass executes only once. CREATE DATABASE;')
```

Comment:
Your task does not require changing this file. If you have found imperfections, please make a pull request upsteram.

Reply:
Sure. Created PR and is already merged: https://github.com/atodorov/qa-automation-python-selenium-101/pull/12 .